clang/tools/3c/README.md

```diff
@@ -53,45 +59,89 @@ of `.c` files and the options for each ([how to do this depends on the
-more source files, for example:
+For simple tests, you can run `3c` on one or more source files without
+a compilation database. For example, the following command will
+convert `foo.c` and print the new version to standard output:
 
 ```
-3c -alltypes -output-postfix=checked foo.c bar.c
+3c -addcr -alltypes foo.c --
 ```
 
-This will write the new version of each file `f.c` to `f.checked.c` in
-the same directory (or `f.h` to `f.checked.h`). If `f.checked.c` would
-be identical to `f.c`, it is not created.
+The `--` at the end of the command line indicates that you _do not_
+want to use a compilation database. This is important to ensure `3c`
+doesn't automatically detect a compilation database that you don't
+want to use. You can add compiler options that you want to use for all
+files after the `--`, such as `-I` for include directories, etc.
 
-The `-alltypes` option causes `3c` to try to infer array types. We
-want to make this the default but haven't done so yet because it
-breaks some things.
+This "stdout mode" only supports a single source file. If you have
+multiple files, you must use one of the modes that writes the output
+to files. You can specify either a string to be inserted into the
+names of the new files or a directory under which they should be
+written. For example, this command:
 
-## More about file handling and compiler options
+```
+3c -addcr -alltypes -output-postfix=checked foo.c bar.c --
+```
 
-As an additional safeguard, `f.checked.c` or `f.checked.h` is not
-written if it is outside the _base directory_, which defaults to the
-working directory but can be overridden with the `-base-dir` option.
-This can help ensure that you don't unintentionally modify external
-libraries (though if their header files are not [annotated for Checked
-C](#annotated-headers), your ability to convert your own program to
-Checked C may be limited). However, there is a bug in the way the base
-directory check handles `..` path components in `-I` directories and
-`#include` paths (terrible, we know!), so until we can fix the bug,
-please avoid using `..` path components (e.g., use `-I` with an
-absolute path instead).
+will write the new versions to `foo.checked.c` and `bar.checked.c`. If
+one of these files is not created, it means the original file needs no
+changes. If `foo.c` includes a header file `foo.h`, then `3c` will
+automatically include it in the conversion and write the new version
+of it to `foo.checked.h` if there are changes; you don't need to
+specify header files separately on the command line, just as they
+wouldn't have their own entries in a compilation database.
 
-You can ignore the errors about a compilation database not being
-found. You can specify a single set of compiler options to use for all
-files by prefixing them with `-extra-arg-before=`, for example:
+The `-output-postfix` mode may be convenient for running tools such as
+`diff` on individual files by hand. For heavier-duty scenarios, this
```

Comment:
What's a "heavier duty scenario" ? I would replace "For heavier-duty scenarios" with "Alternatively"

Reply:
As I implied [here](https://correct-computation.slack.com/archives/G01GKGKHMFD/p1614380211077400?thread_ts=1614374912.062600&cid=G01GKGKHMFD), I was thinking `-output-dir` is better for more advanced processes that handle files in bulk, such as the diff command in the next paragraph.  But we probably don't need to go into this here right now.  Done.